x-pack/test/functional/apps/ml/anomaly_detection/annotations.ts

```diff
@@ -7,96 +7,336 @@
+        await ml.jobAnnotations.assertAnnotationExists({
+          annotation: newText,
+          event: 'user',
+        });
+      });
+
+      it('displays newly created annotation in anomaly explorer and job list', async () => {
+        await ml.testExecution.logTestStep('should display created annotation in anomaly explorer');
+        await ml.navigation.navigateToAnomalyExplorerViaSingleMetricViewer();
+        await ml.anomalyExplorer.assertAnnotationsPanelExists('loaded');
+
+        await ml.jobAnnotations.ensureAnomalyExplorerAnnotationsPanelOpen();
+        await ml.jobAnnotations.assertAnnotationExists({
+          annotation: newText,
+          event: 'user',
+        });
+
+        await ml.testExecution.logTestStep('should display created annotation in job list');
+        await ml.navigation.navigateToJobManagement();
+        await ml.jobTable.waitForJobsToLoad();
+        await ml.jobTable.filterWithSearchString(JOB_CONFIG.job_id, 1);
+        await ml.jobTable.openAnnotationsTab(JOB_CONFIG.job_id);
+        await ml.jobAnnotations.assertAnnotationExists({
+          annotation: newText,
+          event: 'user',
+        });
+      });
+    });
+
+    describe('editing', function () {
+      const annotationId = `edit-annotation-id-${Date.now()}`;
+      const newText = 'Edited annotation text';
+      const expectedOriginalAnnotation = {
+        annotation: annotation.annotation,
+        from: '2016-02-10 22:13:51',
+        to: '2016-02-11 14:24:49',
+        modifiedTime: '2021-07-12 22:38:18',
+        modifiedBy: annotation.modified_username,
+        event: annotation.event,
+      };
+      const expectedEditedAnnotation = {
+        annotation: newText,
+        event: annotation.event,
+      };
+
+      before(async () => {
+        await ml.api.indexAnnotation(annotation, annotationId);
+      });
+
+      it('displays the original annotation correctly', async () => {
+        await ml.testExecution.logTestStep('loads from job list row link');
+        await ml.navigation.navigateToMl();
+        await ml.navigation.navigateToJobManagement();
+
+        await ml.jobTable.waitForJobsToLoad();
+        await ml.jobTable.filterWithSearchString(JOB_CONFIG.job_id, 1);
+        await ml.jobTable.openAnnotationsTab(JOB_CONFIG.job_id);
+        await ml.jobAnnotations.assertAnnotationContentById(
+          annotationId,
+          expectedOriginalAnnotation
+        );
+
+        await ml.jobTable.clickOpenJobInSingleMetricViewerButton(JOB_CONFIG.job_id);
+        await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
+
+        await ml.testExecution.logTestStep('pre-fills the job selection');
+        await ml.jobSelection.assertJobSelection([JOB_CONFIG.job_id]);
+
+        await ml.testExecution.logTestStep('pre-fills the detector input');
+        await ml.singleMetricViewer.assertDetectorInputExist();
+        await ml.singleMetricViewer.assertDetectorInputValue('0');
```

Comment:
See comment above: I think these validations are not needed here.

Reply:
Updated here https://github.com/elastic/kibana/pull/105527/commits/ec07c2c1e48391ce4acbdba4ddc5d8d8fa4593f5